ppmclibs/tinycv_impl.cc

```diff
@@ -438,12 +438,19 @@ void image_map_raw_data(Image *a, const unsigned char *data)
 }
 
 
-void image_map_raw_data_rgb555(Image *a, const unsigned char *data)
+void image_map_raw_data_rgb555(Image *a, const unsigned char *data, bool big_endian)
 {
```

Comment:
as you didn't change the call into this, I assume you broke it. But I suggest the real thing to do is reverting the rgb555 changes.

I will remove the whole function and replace it with a proper call to the _full version once I can test IPMI again.